androbot/handlers.py

```diff
@@ -34,6 +32,8 @@ async def send_start_screen(message: aiotypes.Message):
             reply_markup=view.markup,
         )
 
+        log_event(message.from_user.id, Events.registration, message.text.replace("/start ", ""))
```

Comment:
```suggestion
        log_event(message.from_user.id, Events.Registration, message.text.replace("/start ", ""))
```

Давай добавим заглавные буквы

https://github.com/sf-pet-projects/androbot/blob/f57e8daa7a4d9f40ba550effcb05f9e2dd20d2ff/androbot/types_/event.py#L9